articles/how-to-apply-post-processing.md

```diff
@@ -0,0 +1,120 @@
+# Post Processing in SadConsole
+
+Sometimes you may want to apply a shader or graphical effect to your game when drawing, which this article will explain how to do. 
+
+To apply a post processing effect to the screen, you can override SadConsole's default drawing functionality, and then apply your own custom draw commands.
+
+To do this, you attach a new `DrawableGameComponent` to SadConsole's game instance components, which is set to draw *after* SadConsole has finished creating the frame for the default screen.
+
+This `DrawableGameComponent` should then access SadConsole's final render surface, and apply whatever custom draw code is needed.
+
+## Method
+
+### Setting up the Class
```

Comment:
I would promote these level 3 headers `###` to level 2 headers `##`. All level 2 headers appear in the navigation on the right-side or at the top of the article (based on the view). As you have it right now, the right side will only show

```
- Method
- Troubleshooting
```

I would get rid of Method completely and just have the first header be `Setting up the class` and each header after it the same level. This way the right-side table of contents lists each section. 

Also, headers should not be title cased and should just be sentence cased. For example:

```
Adding the Component, Removing Default Draw

-- Should be 

Adding the component, removing default draw
```

Reply:
I've applied your suggested changes, the titles have been promoted and titlecase has been changed to normal casing.